hack/code/instancetype_testdata_gen/main.go

```diff
@@ -147,13 +147,14 @@ func getInstanceTypeInfo(info *ec2.InstanceTypeInfo) string {
 		fmt.Fprintf(src, "NvmeSupport: aws.String(\"%s\"),\n", lo.FromPtr(info.EbsInfo.NvmeSupport))
 		fmt.Fprintf(src, "},\n")
 	}
-	if info.InferenceAcceleratorInfo != nil {
-		fmt.Fprintf(src, "InferenceAcceleratorInfo: &ec2.InferenceAcceleratorInfo{\n")
-		fmt.Fprintf(src, "Accelerators: []*ec2.InferenceDeviceInfo{\n")
-		for _, elem := range info.InferenceAcceleratorInfo.Accelerators {
-			fmt.Fprintf(src, getInferenceAcceleratorDeviceInfo(elem))
+	if info.NeuronInfo != nil {
```

Comment:
I don't think we do - I think this is now a "legacy" attribute from https://docs.aws.amazon.com/elastic-inference/latest/developerguide/what-is-ei.html 

> I see that Manufacturer is not induced in info.NeuronInfo

Correct - I think it is inferred since Neuron is specific, whereas `gpu` is not - GPU manufacturer could be NVIDIA, AMD, etc.

Reply:
is this comment resolved?